src/Inciter/Carrier.C

```diff
@@ -120,7 +120,7 @@ Carrier::Carrier( const TransporterProxy& transporter,
 //!   mesh chunk we operate on
 //! \param[in] filenodes Map associating old node IDs (as in file) to new node
 //!   IDs (as in producing contiguous-row-id linear system contributions)
-//! \param[in] edgenodemap Map associating new node IDs ('new' as in producing
+//! \param[in] edgenodes Map associating new node IDs ('new' as in producing
 //!   contiguous-row-id linear system contributions) to edges (a pair of old
 //!   node IDs ('old' as in file). These 'new' node IDs are the ones newly
```

Comment:
At multiple points in this file: What does "contiguous-row-id linear" mean? You probably want to rephrase this to make it clear that this is related to the zoltan(?) reordering and partitioning that is going to be passed to hypre and indicate if this is a global ordering (I think yes?)